README.md

```diff
@@ -13,9 +13,16 @@
 - Snippets
 - Syntax Highlighting
 
+### Experimental Features
```

Comment:
Should we tell users how to enable these experimental features?

Reply:
Yes, I'll add a note about enabling the resolver (with the caveat that it only works with limited Chapel code)